bookmarks.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="app-title">
+      <h1>Quiz App</h1>
+    </header>
```

Comment:
```suggestion
    <header class="header">
      <h1 class="header__title">Quiz App</h1>
    </header>
```
Here I would recommend use BEM.